src/styles/app.css

```diff
@@ -36,3 +50,25 @@ h2 a.tutorial-header-link {
 h2:hover a.tutorial-header-link {
   text-decoration: underline;
 }
+
+.qpp-docs-title-container {
+	background: #036;
```

Comment:
Nit but here and below there are 4 spaces of indentation and above we use 2 spaces.